test_script.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+testpath="geopyspark/tests/"
+testpattern="*_test.py"
+
+for f in $testpath$testpattern $testpath/**/$testpattern;
```

Comment:
Do `$testpath$testpattern` and `$testpath/**/$testpattern` produce the same list or different ones?

Assuming that these two statement are not redundant, :+1: